### ISSUE TITLE
Reset RearViewController

### ISSUE THREAD

**Issue by GuyFran** [John-Lluch/SWRevealViewController#360]
Hi.
What is the expected way to reset the component with a nil rearviewcontroller ?

On init, it seems that as long as I didn't set my own rearviewcontroller, revealtoggle isn't triggered.

After I set, it's obviously revealing the rearviewcontroller.

But now the only way I've found to "remove" my previously set rearviewcontroller, is to call setrearviewcontroller with a blank UIViewcontroller. But this would not disable the animation upon "Revealtoggle" call anymore ?

Basically, what's the step to reset the whole thing ?

Thx

**Comment by MobileMon** [comment]
I have the same problem, have you found a solution?

**Comment by MobileMon** [comment]
Because i was calling the remove in viewWillDisappear, it meant self.revealViewController was already nil so this fixes it:

    AppDelegate *delegate = (AppDelegate *)[UIApplication sharedApplication].delegate;
    SWRevealViewController *revealController = (SWRevealViewController*)delegate.window.rootViewController;
